.changeset/nervous-mirrors-dress.md

```diff
@@ -0,0 +1,20 @@
+---
+'@guardian/source-react-components-development-kitchen': major
+---
+
+Refactor `Tabs` so it can receive an optional theme. This will require consumers
+to update their calls, but it should be compatible with the existing exported
+light and dark themes.
```

Comment:
same as previously, is it possible to add more details about the new requirement?